retriever/__main__.py

```diff
@@ -62,16 +62,15 @@ def main():
             return
 
         if args.command == "citation":
-            if args.dataset is None:
+            citations = get_script_citation(args.dataset)
+            if not citations:
+                return
+            if citations == [VERSION]:
                 print("\nCitation for retriever:\n")
                 print(CITATION)
             else:
-                scripts = name_matches(script_list, args.dataset)
-                for dataset in scripts:
-                    print("\nDataset:  {}".format(dataset.name))
-                    print("Citation:   {}".format(dataset.citation))
-                    print("Description:   {}\n".format(dataset.description))
-
+                for citation in citations:
+                    print("Citation:   {}".format(citation))
```

Comment:
Moving forward, I think we can start making use of python `f-strings`. https://www.python.org/dev/peps/pep-0498/
The above statement can be written as:
```python
print(f"Citation:   {citation}")
```

Reply:
Thanks for your guide. I found that many expression in __main__.py is like  `print("Citation:   {}".format(citation))`. Do I need to change all of the expression to the example you provided? Thank you!

Reply:
Not for this PR. you can change those others  print statements in a different PR.
Let this PR have only the related changes to the issue. i.e citation related

Reply:
Ok, I will modify the code.

Reply:
Before I pull request, I want to know how can I only use AppVeyor test like #1434 
Thank you！

Reply:
You can use only Appveyor for test. I am trying to make tests on linux work again

Reply:
Does Travis CI build work now? Is there any approach to test the code in this pr again without additional commit and push?

Reply:
Have you tried the Docker setup. We have docs on testing using docker

Reply:
Ok, I will try to use docker. Thanks!